src/controllers/vscodeWrapper.ts

```diff
@@ -339,6 +339,13 @@ export default class VscodeWrapper {
     }
 
     /**
+     * Change a configuration setting
+     */
+    public setConfiguration(extensionName: string, resource: string, value: any): Thenable<void> {
```

Comment:
Why is this a thenable instead of just using promises?

Reply:
That comes from the vscode api